src/js/actions/contactActions.js

```diff
@@ -1,21 +1,46 @@
-import webAPI from '../utils/webAPI';
+import actionTypes from "../constants/actionTypes"
+import webAPI from '../utils/webAPI'
 import validator from '../utils/validator';
+import { receiveError } from './errorActions'
 
-const contact = (data) => {
-  const {isValid, concatenatedErrors} = validator.validate(data);
-  if (isValid) {
-    // TODO: Use contact us endpoint.
-    // TODO: Use this error message on server error: Something prevented your message from submitting successfully. Please try again later.'
-    return new Promise(function(resolve, reject) {
-      resolve();
-    });
-  } else {
-    return new Promise(function(resolve, reject) {
-      reject(concatenatedErrors);
-    });
+export const sendContactActions = () => {
+  return {
+    request: (data) => {
+      return {
+        type: actionTypes.REQUEST_SEND_FEEDBACK,
+        payload: { data }
+      }
+    },
+    receive: (data) => {
+      return {
+        type: actionTypes.RECEIVE_SEND_FEEDBACK,
+        payload: { receivedAt: new Date() }
+      }
+    },
+    fail: (errors) => {
+      return {
+        type: actionTypes.FAIL_SEND_FEEDBACK,
+        payload: { errors, receivedAt: new Date() }
+      }
+    }
+  }
+}
+
+export const sendFeedback = (data) => {
+  let actions = sendContactActions()
+  return function(dispatch) {
+    dispatch(actions.request(data))
```

Comment:
Variable naming. `contactActions` would be a better description than `actions`. Halfway into the code I forgot what `actions` we were talking about.